FAQ.md

```diff
@@ -54,10 +54,18 @@ Essentially, anything which provides the full identity of what you want to simul
 
 If you are unable to provide a molecule in the formats recommended above and want to attempt to infer the bond orders and atomic formal charges, there are tools available elsewhere that can provide guesses for this problem. These tools are not perfect, and the inference problem itself is poorly defined, so you should review each output closely (see our [Core Concepts](users/concepts) for an explanation of what information is needed to construct an OpenFF Molecule). Some tools we know of include:
 
-- the OpenEye toolkits' [`OEPerceiveBondOrders`](https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemFunctions/OEPerceiveBondOrders.html) functionality
+- the OpenEye Toolkits' [`OEPerceiveBondOrders`](https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemFunctions/OEPerceiveBondOrders.html) functionality
```

Comment:
I take it to be plural when phrased this way because that's how they communicate it (https://docs.eyesopen.com/toolkits/python/index.html, https://anaconda.org/openeye/openeye-toolkits) as I'm not sure a single toolkit from them exists. I'm also not sure where one toolkit begins and the other ends ...

Reply:
Ok yeah good call 🙂